docs/future/vite.md

```diff
@@ -338,6 +340,8 @@ npm install -D vite
 
 Remix is now just a Vite plugin, so you'll need to hook it up to Vite.
 
+**Note**: Remix requires a minimum Vite version of 5.0.0 to work properly.
```

Comment:
We already specify [Vite v5.1.0 as a minimal version in our peer dependencies](https://github.com/remix-run/remix/blob/main/packages/remix-dev/package.json#L100) as a single-source-of-truth.